### PR TITLE
Issue #321, fix 404 links and adjust readme order in demo classical_c…

### PR DESCRIPTION
**Pull request by QianyeYang**
…t_headneck_affine.

# Description

Please include a summary of the change and which issue is fixed. Please also include
relevant motivation and context. List any dependencies that are required for this
change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not
      work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so
we can reproduce. Please also list any relevant details for your test configuration If
there are no tests written, flag with label so we can help generate some tests before
merge.

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Firmware version:
- Hardware:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: demos/classical_ct_headneck_affine/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Classical Affine Registration for Head-and-Neck CT Images
 
 > **Note**: Please read the
-> [DeepReg Demo Disclaimer](introduction.html#demo-disclaimer).
+> [DeepReg Demo Disclaimer](https://deepreg.readthedocs.io/en/325-improve-contributing-pages/demo/introduction.html#demo-disclaimer).
```
Comment (mathpluscode) [inline demos/classical_ct_headneck_affine/README.md:4]: why this change? there should be no 404 without this change

**Review by mathpluscode — CHANGES_REQUESTED** [review]
The change of link is strange ;)

File: demos/classical_ct_headneck_affine/README.md (hunk 1 above)
Comment (QianyeYang) [reply demos/classical_ct_headneck_affine/README.md:4]: > 
> 
> why this change? there should be no 404 without this change

Did you click on that? Still not works for me.

Comment (mathpluscode) [reply demos/classical_ct_headneck_affine/README.md:4]: how you tested this and found it buggy? to test the docs, either you do it locally, either we activate the branch on rtd

**Review by mathpluscode — APPROVED** [review]
